advanced/am-pattern-3/requirements.yaml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 dependencies:
   - name: mysql-am
-    version: "3.1.0-2"
+    version: "3.1.0-1"
```

Comment:
@Chathuru the upcoming release is `3.1.0-2`. So, no need to change.

Reply:
Update to the new dependency version in all 3 patterns in https://github.com/wso2/kubernetes-apim/pull/365/commits/656646538e8cc1b8d8fbc51cb3d206a1c58101e5